hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatmap.tsx

```diff
@@ -51,11 +51,62 @@ interface ITreeState {
   entityType: string;
   date: string;
   treeEndpointFailed: boolean;
+  inputPathValid: inputPathValidity;
+  helpMessage: string;
 }
 
 let minSize = Infinity;
 let maxSize = 0;
 
+const colourScheme = {
+  pastel_greens: [
```

Comment:
I am not sure when this green shades will appear, as there are 3 categories, the access of volume/buckets/keys, may fall under like Most Frequently accessed (MFA - Red), Frequently Accessed (FA - Orange), Least Accessed (LA - Yellow). Under which category and how this shade of green will appear ?

Reply:
So I had kept this code in place, if later we wanted to shift to the alternate colour theme, this way we could handle the change easier, but I will remove this since it is redundant code.